locales/pt-BR.schema.json

```
@@ -842,6 +871,12 @@
         },
         "paragraph": {
           "content": "Para alterar resumos, edite os [posts do blog].(https://help.shopify.com/pt-BR/manual/online-store/blogs/writing-blogs#exiba-o-resumo-de-um-post)"
```

Not part of this PR, but this will generate a broken link. 
```suggestion
          "content": "Para alterar resumos, edite os [posts do blog](https://help.shopify.com/pt-BR/manual/online-store/blogs/writing-blogs#exiba-o-resumo-de-um-post)."
```

---

I've fixed this in the Smartling backend